templates/teddycloud.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<Container version="2">
+    <Name>teddycloud</Name>
+    <Repository>ghcr.io/toniebox-reverse-engineering/teddycloud:latest</Repository>
+    <Registry>ghcr.io/toniebox-reverse-engineering/teddycloud:latest</Registry>
+    <Network>bridge</Network>
+    <Privileged>false</Privileged>
+    <Support>https://t.me/toniebox_reverse_engineering</Support>
+    <Project>https://github.com/toniebox-reverse-engineering/teddycloud</Project>
+    <Overview>teddyCloud is an alternative server for your Toniebox, allowing you to host the cloud services locally. This gives you the control about which data is sent to the original manufacturers cloud and allows you to host your own figurine audio files on e.g. your NAS or any other server.
+    </Overview>
+    <WebUI>http://[IP]:[PORT:7780]</WebUI>
+    <Category>MediaServer:Music MediaServer:Other</Category>
+    <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/teddycloud.xml</TemplateURL>
+    <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/teddycloud.xml</Icon>
+    <Config Name="port 80 (optional)" Target="80" Default="" Mode="tcp" Description="optional" Type="Port" Display="always" Required="false" Mask="false">7780</Config>
+    <Config Name="443" Target="443" Default="" Mode="tcp" Description="" Type="Port" Display="always-hide" Required="true" Mask="false">7443</Config>
```

Comment:
I think both these ports should be always visible and required. Also, please set their `Default`s and use more descriptive `Name`s.